testing/scenario_app/README.md

```diff
@@ -72,5 +81,5 @@ To generate new lockfiles, run:
 
 ```bash
 cd android
-../../../../gradle/bin/gradle generateLockfiles
+gradle generateLockfiles
```

Comment:
Please point to the gradle we supply with the engine - we should not be using arbitrary versions of gradle from the system, since we're not testing them on CI.